ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_models.robot

```diff
@@ -84,9 +118,10 @@ Setup Test Variables
     ELSE
         Set Test Variable    ${use_port_forwarding}    ${FALSE}
     END
-    
-    
+
 Start Port-forwarding
     [Arguments]    ${namespace}    ${model_name}
     ${process}=    Start Process    oc -n ${namespace} port-forward svc/${model_name}-predictor 8033:80
     ...    alias=llm-query-process    stderr=STDOUT    shell=yes
+    Process Should Be Running    ${process}
+    sleep  5s
```

Comment:
why sleeping? was this failing for you?

Reply:
yes it was constantly failing for me  and normally it is always good to wait because port forwarding take some second to get started and this one will make port forwarding more stable in that regard